From af870d2e32699963b218ff33c25a2d4452a75be8 Mon Sep 17 00:00:00 2001
From: renatofilho <renato.filho@openbossa.org>
Date: Mon, 18 Oct 2010 19:01:04 -0300
Subject: [PATCH] Fixed PySide plugin for uiloader dependency.

Reviewer: Hugo Parente Lima <hugo.pl@gmail.com>
          Luciano Wolf <luciano.wolf@openbossa.org>
---
 CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4ad006e..dc8f234 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -183,7 +183,7 @@ set(GENERATOR_EXTRA_FLAGS --generatorSet=shiboken --enable-parent-ctor-heuristic
 enable_testing()
 
 add_subdirectory(libpyside)
-if(QT_QTDESIGNER_FOUND)
+if(QT_QTUITOOLS_FOUND AND QT_QTDESIGNER_FOUND)
     add_subdirectory(plugins)
 endif()
 # project directories
-- 
1.6.1

